addon/ng2/blueprints/ng2/files/__path__/manifest.webapp

```diff
@@ -0,0 +1,14 @@
+{
+  "short_name": "<%= htmlComponentName %>",
```

Comment:
short_name should still be human readable. If CLI doesn't have something appropriate, just omit it.